WordPress/Classes/ViewRelated/Stats/Today Widgets/Data/AllTimeWidgetStats.swift

```diff
@@ -34,7 +34,20 @@ extension AllTimeWidgetStats {
             return try decoder.decode(AllTimeWidgetStats.self, from: data)
         } catch {
             DDLogError("Failed loading AllTimeWidgetStats data: \(error.localizedDescription)")
-            return AllTimeWidgetStats()
+            return nil
+        }
+    }
+
+    static func clearSavedData() {
+        guard let dataFileURL = AllTimeWidgetStats.dataFileURL else {
+            return
```

Comment:
Is it worth adding any kind of logging here to track when the file URL was empty?